### PR TITLE
fix: DEV-3389: Display indices in the regions list

### PR DESCRIPTION
**Pull request by nicholasrq**

### CODE REVIEW

File: src/components/SidePanels/OutlinerPanel/OutlinerTree.tsx [hunk 1]
```diff
@@ -37,6 +37,8 @@ const OutlinerTreeComponent: FC<OutlinerTreeProps> = ({
   const eventHandlers = useEventHandlers({ regions, onHover });
   const regionsTree = useDataTree({ regions, hovered, rootClass, selectedKeys });
 
+  console.log({ regionsTree });
```
Comment (bmartel) [inline src/components/SidePanels/OutlinerPanel/OutlinerTree.tsx:40]: leftover console 
```suggestion
```

File: src/components/SidePanels/OutlinerPanel/OutlinerTree.tsx [hunk 2]
```diff
@@ -242,9 +244,12 @@ const RootTitle: FC<any> = observer(({
     setCollapsed(!collapsed);
   }, [collapsed]);
 
+  console.log(props.entity);
```
Comment (bmartel) [inline src/components/SidePanels/OutlinerPanel/OutlinerTree.tsx:247]:
```suggestion
```

**Review by bmartel — CHANGES_REQUESTED** [review]
couple leftover consoles, implementation looks good

File: src/components/SidePanels/OutlinerPanel/OutlinerTree.tsx [hunk 3]
```diff
@@ -51,7 +53,7 @@ const OutlinerTreeComponent: FC<OutlinerTreeProps> = ({
           className={rootClass.toClassName()}
           treeData={regionsTree}
           selectedKeys={selectedKeys}
-          icon={({ entity }: any) => <NodeIconComponent node={entity}/>}
+          icon={({ entity, isNotLabel }: any) => <NodeIconComponent node={entity}/>}
```
Comment (hlomzik) [inline src/components/SidePanels/OutlinerPanel/OutlinerTree.tsx:56]: not used

File: src/stores/RegionStore.js [hunk 4]
```diff
@@ -351,6 +350,7 @@ export default types.model("RegionStore", {
           key,
           isArea: false,
           children: [],
+          isNotLabel: true,
```
Comment (hlomzik) [inline src/stores/RegionStore.js:353]: what's this? is it always true? or is it only for "group by label" mode? so I'd suggest to rename it

**Review by hlomzik — CHANGES_REQUESTED** [review]

**Review by bmartel — APPROVED** [review]

**Pull request merged**